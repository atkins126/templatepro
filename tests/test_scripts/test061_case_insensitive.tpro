{{:VALUE2|sayhello}}
{{:VALUE2}} => {{:VALUE2 | lowercase}}
{{:VALUEDATE|formatdatetime,"yyyy-mm-dd"}}
{{for obj in OBJECTS}}
  {{:OBJ.Prop1 | lpad, 10 }}
{{endfor}}

{{for cUsT in cUsToMeRs}}
Customer: {{:cUsT.Name|lpad, 10, "_"}}
{{for obj in objects}}
{{:obj.Prop1|lpad, 10}}
{{endfor}}  
{{endfor}}
{{if VALUE2|sayhello }}
filter in if expression
{{endif}}