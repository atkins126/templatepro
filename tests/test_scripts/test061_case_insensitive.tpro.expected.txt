Hello DANIELE2
DANIELE2 => daniele2
2024-08-20
    value1.1
    value1.2
    value1.3

Customer: ______Ford
  value1.1
  value1.2
  value1.3
  
Customer: ___Ferrari
  value1.1
  value1.2
  value1.3
  
Customer: _____Lotus
  value1.1
  value1.2
  value1.3
  
Customer: _______FCA
  value1.1
  value1.2
  value1.3
  
Customer: ___Hyundai
  value1.1
  value1.2
  value1.3
  
Customer: _De Tomaso
  value1.1
  value1.2
  value1.3
  
Customer: _____Dodge
  value1.1
  value1.2
  value1.3
  
Customer: _____Tesla
  value1.1
  value1.2
  value1.3
  
Customer: _______Kia
  value1.1
  value1.2
  value1.3
  
Customer: ______Tata
  value1.1
  value1.2
  value1.3
  
Customer: Volkswagen
  value1.1
  value1.2
  value1.3
  
Customer: ______Audi
  value1.1
  value1.2
  value1.3
  
Customer: _____Skoda
  value1.1
  value1.2
  value1.3
  
filter in if expression
